packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -206,6 +209,15 @@ export class CeramicCliUtils {
       if (syncOverride) {
         config.node.syncOverride = syncOverride
       }
+
+      config.indexing.composedbEnabled = fromBooleanInput(enableComposedb, true)
+      if (process.env.CERAMIC_ENABLE_COMPOSE_DB) {
```

Comment:
since composedb should be enabled by default, should this env var be `CERAMIC_DISABLE_COMPOSE_DB`?